src/components/schema-tree.js

```diff
@@ -244,7 +244,12 @@ export default class SchemaTree extends LitElement {
                 : ''
           }
           <span class="${dataTypeCss}">
-            ${dataType === 'array' ? `${type}[]` : `${type}`}
+            ${dataType === 'array'
```

Comment:
Let's not change the format from `type[]` to `[type]`, so let's revert that change.

Reply:
why? how i will notice if this is array?

Reply:
What do you mean? `type` is a type and `type[]` is an array.

Reply:
if i correct, this part will be similar to swagger-ui design, and it will more readable

Reply:
Let's not make visual changes just because "we like swagger-ui" better. If we want to change the display in a later PR we can do that, let's fix the array/format display here.

Reply:
if we did like swagger-ui we use swagger-ui. But we try to finish one of fork that will be useful from point of documentation usability. If you think we are in the wrong fork, that ok. You are the maintainer, this is your rights.

Reply:
It's a problem of understanding. It sounds like you are saying "We should change this because Swagger-ui does it this way", which doesn't help me understand the problem. It's possible there is an issue with our current solution, just help me understand what that is.

It seems like you are suggesting `[type]` is more readable than `type[]`. Perhaps it is, I guess from a naive perspective one could say, I don't see the difference. Which makes it subjective. If this change is what sells using this library for you, help us understand how to justify one display over the other. We want to pick the best one, objectively.

Reply:
For me, it looks like you self never use your tool. Can you notice a difference?
![Screen Shot 2022-09-02 at 3 01 51 PM](https://user-images.githubusercontent.com/28315000/188227856-43e686bc-1792-4046-90ca-03a5fa8f8ca1.png)
![Screen Shot 2022-09-02 at 3 04 01 PM](https://user-images.githubusercontent.com/28315000/188228182-b1c920ac-473f-4f4b-bca8-507f1406de83.png)

Reply:
and simple way to check changes, just switch to branch, build and check in browser

Reply:
I had a talk with our UX team, let's go the `[int64]`, but the `[` and `]` should be white/default color, and then the `int64` color can be left to be blue as defined by the `integer` class.

Make sense?